src/data/nat/factorization/prime_pow.lean

```diff
@@ -60,6 +60,31 @@ lemma is_prime_pow_iff_card_support_factorization_eq_one {n : ℕ} :
 by simp_rw [is_prime_pow_iff_factorization_eq_single, finsupp.card_support_eq_one', exists_prop,
   pos_iff_ne_zero]
 
+lemma exists_ord_compl_eq_one_of_is_prime_pow {n : ℕ} (h : is_prime_pow n) :
```

Comment:
Could you add the `iff` version as well (assuming `n ≠ 1`), since `is_prime_pow_of_ord_compl_eq_one` is almost the converse?

Reply:
Ok, I've replaced `is_prime_pow_of_ord_compl_eq_one` with the `iff` version of `exists_ord_compl_eq_one_of_is_prime_pow`.